docs/src/main/asciidoc/cache.adoc

```diff
@@ -522,6 +522,67 @@ quarkus.cache.caffeine."bar".maximum-size=1000 <2>
 <1> The `foo` cache is being configured.
 <2> The `bar` cache is being configured.
 
+== Enabling Micrometer metrics
+
+Each cache declared using the <<#annotations-api,annotations caching API>> can be monitored using Micrometer metrics.
+
+[NOTE]
+====
+The cache metrics collection will only work if your application depends on a `quarkus-micrometer-registry-*` extension.
+See the <<micrometer.adoc,Micrometer metrics guide>> to learn how to use Micrometer in Quarkus.
+====
+
+The cache metrics collection is disabled by default.
+It can be enabled from the `application.properties` file:
+
+[source,properties]
+----
+quarkus.cache.caffeine."foo".metrics.enabled=true
+----
+
+[WARNING]
+====
+Like all instrumentation methods, collecting metrics comes with a small overhead that can impact the application performances.
+====
+
+The collected metrics contain cache statistics such as:
+
+- the approximate current number of entries in the cache
+- the number of entries that were added to the cache
+- the number of times a cache lookup has been performed, including information about hits and misses
+- the number of evictions and the weight of the evicted entries
+
+Tags can be added to these metrics.
+This is also configured from the `application.properties` file:
+
+[source,properties]
+----
+quarkus.cache.caffeine."foo".metrics.tags=tag1,value1,tag2,value2 <1>
```

Comment:
we did it this way elsewhere? Because it looks like it could use a Map :)

Reply:
The [Micrometer API](https://javadoc.io/doc/io.micrometer/micrometer-core/1.0.3/io/micrometer/core/instrument/binder/cache/CaffeineCacheMetrics.html#monitor-io.micrometer.core.instrument.MeterRegistry-C-java.lang.String-java.lang.String...-) accepts a vararg for the tags which is why I used a `String[]` configuration value. Using a `Map` and translating it back to a vararg would probably make things clearer for the Quarkus user indeed. I'll see what I can do.

Reply:
y.. I have avoided declaring tags via config and have recommended using MeterFilters instead (which can be enabled by profile, or use injected config values, etc).

None of this code needs to deal with, remember, or add common tags in that case. A user-defined MeterFilter will do it:
https://quarkus.io/guides/micrometer#using-meterfilter-to-configure-metrics

Reply:
I removed the `tags` configuration value.